01_Data/src/layers/sequelize/repository/TransactionEvent.ts

```diff
@@ -79,18 +79,21 @@ export class SequelizeTransactionEventRepository extends SequelizeRepository<Tra
 
       const updatedTransaction = Transaction.build({
         stationId,
-        isActive: value.eventType !== TransactionEventEnumType.Ended,
```

Comment:
Hi Thana, why this way doesn't work?

Reply:
the default value on the column overwrote this logic. Modifying the column was enough to fix the immediate issue, but I also wanted to make sure that in the case where messages are received out of order the Transaction stays inactive no matter what, so I refactored this logic as well.